packages/eslint-config-small-ads/index.js

```diff
@@ -21,23 +21,11 @@ module.exports = {
     sourceType: "module",
   },
   plugins: ["react-hooks", "@typescript-eslint/eslint-plugin"],
-  rules: {
-    "import/prefer-default-export": "off",
```

Comment:
Hey, we want this one to be global _but_ the `"import/no-extraneous-dependencies": "off",` is specific to the `ui`

Reply:
Oh, I saw I had to use it in both projects, but I wasn't not sure if we would need it in *every* project in case we create something new. But it makes sense that we have it global for now if we are using it in every project. Total sense.

Reply:
This is something it is in my head. And you couldnt really guess it :) So no stress, I should have explained myself better